src/css/select.less

```diff
@@ -23,6 +24,13 @@
   position: relative;
   display: inline-block;
   outline: none;
+
+  &:focus {
+    .ember-select-choice {
```

Comment:
can we put these on the same line, so it reads: "&:focus .ember-select-choice"? It makes it easier to read.

Thanks!